.mocharc.jsonc

```diff
@@ -10,6 +10,6 @@
   // "grep": ["some test title here"],
   "exit": true,
   "check-leaks": true,
-  "timeout": 30000,
+  "timeout": 1200000,
```

Comment:
Are the tests on DB2 that much slower? Maybe it would be better to keep the default on 30000 but specify a different value in the command line when executing the script for DB2. That takes priority according to https://mochajs.org/#merging

Reply:
Done.